locals.tf

```diff
@@ -1,3 +1,46 @@
 locals {
-  
+    path = "${path.module}/queries"
+
+    law_name = reverse(split("/", var.law_id))[0]
+    law_rg   = split("/", var.law_id)[4]
```

Comment:
As you also need the location of the workspace it is more straight forward to just request the needed information of the law as an object instead of single variables. 
```suggestion
```